node/tools/go.mod

```diff
@@ -1,28 +1,5 @@
 module github.com/certusone/wormhole/node/tools
 
-go 1.19
+go 1.16
```

Comment:
Why are we rolling back the version of go? If this is intentional, please add a comment.

Reply:
This should not be necessary and fail with >1.19. Also why are all the indirects dropped?

Reply:
Idk. With go 1.19 I get a build error. 

How do I generate the indirects?

Reply:
`go mod tidy`